src/App/ComponentsDocumentation/components/Select/index.js

```diff
@@ -58,7 +58,7 @@ const ContentGuidelines = () => (
         <h2 id="content-guidelines">Content guidelines</h2>
 
         <div className="d-flex justify-content-center slab slab-plain px-5 pt-5 pb-4 no-pointer-events">
```

Comment:
I was wondering if we should remove the `no-pointer-events` here 🤔 
Maybe have a quick discussion with Maja about it.

Reply:
Removed no-pointer-events and added "related option" as options

Reply:
OK, let's wait until Maja can answer :)